cmd/crates/soroban-rpc/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
```

Comment:
IMO this crate should live in the new `soroban-rpc` repo, but it should be easy to move it over once this PR is merged

Reply:
I agree. Perhaps it's best to move it after the first publish?

Reply:
Updated deps to not use workspace to make this transition easier.